docs/dotnet/dotnet-signalr-metrics.md

```diff
@@ -0,0 +1,68 @@
+# Semantic Conventions for SignalR metrics emitted by ASP.NET Core
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions for SignalR metrics emitted by .NET components and runtime.
+
+**Disclaimer:** These are initial .NET metric instruments available in .NET 8 but more may be added in the future.
+
+<!-- toc -->
+
+- [Metric: `signalr_http_transport.connection.duration`](#metric-signalr_http_transportconnectionduration)
+- [Metric: `signalr_http_transport.active_connections`](#metric-signalr_http_transportactive_connections)
+
+<!-- tocstop -->
+
+## Metric: `signalr_http_transport.connection.duration`
+
+**Status**: [Experimental][DocumentStatus]
+
+This metric is required.
+
+<!-- semconv metric.signalr.server.connection.duration(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `signalr_http_transport.connection.duration` | Histogram | `s` | The duration of connections on the server. [1] |
+
+**[1]:** **TODO: why http_transport? can we report protocol name as dimension and not a metric name?**
+**TODO: we should do signalr.server**
+Corresponding `EventCounter` name is `connections-duration`; Meter name is `Microsoft.AspNetCore.Http.Connections`
+<!-- endsemconv -->
+
+<!-- semconv metric.signalr.server.connection.duration(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `signalr.http_transport.status` | string | TODO [1] | `TODO` | Recommended |
```

Comment:
Porting active comment threads from the word doc:

## Trask Stalnaker
Any reason not to use signalr.http_transport.*? (making signalr a namespace, similar to kestrel)